code/modules/mob/living/carbon/xenomorph/attack_alien.dm

```diff
@@ -45,64 +45,78 @@
 		return
 	Paralyze(16 SECONDS)
 
-/mob/living/carbon/human/attack_alien_disarm(mob/living/carbon/xenomorph/X, dam_bonus)
-	var/tackle_pain = X.xeno_caste.tackle_damage
-	if(!tackle_pain)
-		return FALSE
-	if(isnestedhost(src)) //No more memeing nested and infected hosts
-		to_chat(X, "<span class='xenodanger'>We reconsider our mean-spirited bullying of the pregnant, secured host.</span>")
+/mob/living/carbon/human/attack_alien_disarm(mob/living/carbon/xenomorph/X, dam_bonus, set_location = FALSE, random_location = FALSE, no_head = FALSE, no_crit = FALSE, force_intent = null)
+	if(!can_xeno_slash(X))
```

Comment:
copypaste
either
 A) Remove alien disar, since nothing uses it (?)
 B) just have tackle damage hugely reduced to the point of uselessness

Reply:
I did option 1